src/main/resources/csar/scripts/kubectl_statefulset_delete.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# configuration
+KUBE_ADMIN_CONFIG_PATH=/etc/kubernetes/admin.conf
+
+# provided variables:
+# KUBE_STATEFULSET_ID: contains the k8s statefulset id to undeploy
+
+NAMESPACE_OPTION=""
+if [ ! -z "$NAMESPACE" ]; then
+   NAMESPACE_OPTION="-n $NAMESPACE "
+fi
+
+function undeploy_resource(){
+    # undeploy
+
+    kubectl --kubeconfig "${KUBE_ADMIN_CONFIG_PATH}" ${NAMESPACE_OPTION}delete statefulset "${KUBE_STATEFULSET_ID}"
```

Comment:
Should quote variables

```suggestion
    kubectl --kubeconfig "${KUBE_ADMIN_CONFIG_PATH}" "${NAMESPACE_OPTION}delete" statefulset "${KUBE_STATEFULSET_ID}"
```